docker/sonarqube/sonar.properties

```diff
@@ -0,0 +1,377 @@
+
+# Maximum number of files to keep if a rolling policy is enabled.
+#    - maximum value is 20 on size rolling policy
+#    - unlimited on time rolling policy. Set to zero to disable old file purging.
+#sonar.log.maxFiles=7
+
+# Access log is the list of all the HTTP requests received by server. If enabled, it is stored
+# in the file {sonar.path.logs}/access.log. This file follows the same rolling policy as other log file
+# (see sonar.log.rollingPolicy and sonar.log.maxFiles).
+#sonar.web.accessLogs.enable=true
+
+# Format of access log. It is ignored if sonar.web.accessLogs.enable=false. Possible values are:
+#    - "common" is the Common Log Format, shortcut to: %h %l %u %user %date "%r" %s %b
+#    - "combined" is another format widely recognized, shortcut to: %h %l %u [%t] "%r" %s %b "%i{Referer}" "%i{User-Agent}"
+#    - else a custom pattern. See http://logback.qos.ch/manual/layouts.html#AccessPatternLayout.
+# The login of authenticated user is not implemented with "%u" but with "%reqAttribute{LOGIN}" (since version 6.1).
+# The value displayed for anonymous users is "-".
+# The SonarQube's HTTP request ID can be added to the pattern with "%reqAttribute{ID}" (since version 6.2).
+# If SonarQube is behind a reverse proxy, then the following value allows to display the correct remote IP address:
+#sonar.web.accessLogs.pattern=%i{X-Forwarded-For} %l %u [%t] "%r" %s %b "%i{Referer}" "%i{User-Agent}" "%reqAttribute{ID}"
+# Default value (which was "combined" before version 6.2) is equivalent to "combined + SQ HTTP request ID":
+#sonar.web.accessLogs.pattern=%h %l %u [%t] "%r" %s %b "%i{Referer}" "%i{User-Agent}" "%reqAttribute{ID}"
+
+
+#--------------------------------------------------------------------------------------------------
+# OTHERS
+
+# Delay in seconds between processing of notification queue. Default is 60 seconds.
+#sonar.notifications.delay=60
+
+# Paths to persistent data files (embedded database and search index) and temporary files.
+# Can be absolute or relative to installation directory.
+# Defaults are respectively <installation home>/data and <installation home>/temp
+#sonar.path.data=data
+#sonar.path.temp=temp
+
+
+#--------------------------------------------------------------------------------------------------
+# DEVELOPMENT - only for developers
+# The following properties MUST NOT be used in production environments.
+
+# Dev mode allows to reload web sources on changes and to restart server when new versions
+# of plugins are deployed.
+#sonar.web.dev=false
+
+# Elasticsearch HTTP connector, for example for KOPF:
+# http://lmenezes.com/elasticsearch-kopf/?location=http://localhost:9010
+#sonar.search.httpPort=-1
+
+sonar.log.console=true
+sonar.jdbc.username=${env:JDBC_USERNAME}
+sonar.jdbc.password=${env:JDBC_PASSWORD}
+sonar.jdbc.url=${env:JDBC_URL}
+ldap.bindDn=${env:LDAP_BINDDN}
+ldap.bindPassword=${env:LDAP_BINDPASSWD}
+ldap.url=${env:LDAP_URL}
+ldap.realm=${env:LDAP_REALM}

```

Comment:
This one was "fixed" by Kevin a few days ago - please ensure it carries over:
https://github.com/rht-labs/openshift-sonarqube/pull/17/files